quarto/src/Entity/Game.php

```diff
@@ -460,10 +462,12 @@ public function getWinningPieces() : array
             if ($gridSize > 0) {
```

Comment:
I don't understand why you test the grid size at each loop instead of testing it before the foreach

Reply:
me too ;)